px-vis-xy-chart.html

```diff
@@ -153,13 +154,22 @@
             config='{"zoom": true, "pan": true, "tooltip": true}'>
           </px-vis-toolbar>
           <!-- Core -->
+          <px-threshold-bar
```

Comment:
By permanently adding px-threshold-bar we are adding a dependency to this particular component - do you think that we could utilize the [Slot element](https://polymer-library.polymer-project.org/2.0/docs/devguide/shadow-dom#slot-apis) (supported in Polymer 2, 3 and is in the Web Components API) here to render whichever element we wanted to?